### PR TITLE
Replacing date_format_medium_no_time with hook_date_format_types

### PR DESCRIPTION
**Pull request by bitamar**

### CODE REVIEW

File: garmentbox/modules/custom/garmentbox_general/garmentbox_general.module [hunk 1]
```diff
@@ -402,3 +402,27 @@ function _garmentbox_general_get_node_page_primary_button($node) {
       return $result;
   }
 }
+
+/**
+ * Implements hook_date_format_types()
```
Comment (barochoren) [inline garmentbox/modules/custom/garmentbox_general/garmentbox_general.module]: Missing dot.

File: garmentbox/garmentbox.install [hunk 2]
```diff
@@ -31,7 +31,6 @@ function garmentbox_install() {
   variable_set('date_default_timezone', 'Asia/Jerusalem');
   variable_set('date_first_day', 1);
   variable_set('date_format_medium', 'D, m/d/Y H:i');
```
Comment (amitaibu) [inline garmentbox/garmentbox.install]: Also this variable can be replaced.

Comment (bitamar) [inline garmentbox/garmentbox.install]: Done

File: garmentbox/modules/custom/garmentbox_general/garmentbox_general.module (hunk 1 above)
Comment (bitamar) [inline garmentbox/modules/custom/garmentbox_general/garmentbox_general.module]: Fixed

**Comment by amitaibu** [comment]
Merged.

**Pull request closed without merging**